Dockerfile

```diff
@@ -19,4 +19,6 @@ RUN pip install -r $REQUIREMENTS --disable-pip-version-check
 
 ADD . /app
 
-EXPOSE 8000
\ No newline at end of file
+EXPOSE 8000
+
+CMD /bin/sh -c "./wait-for-it.sh -t $WAIT_FOR_IT_TIMER $DATABASE_HOST:$DATABASE_PORT -- app/manage.py migrate && app/manage.py && uwsgi"
```

Comment:
There's a `manage.py` in there without parameter.. might wanna remove that :)

Reply:
> something like that?

Much better :) Did you test it, did it build / start correctly?

Reply:
it did build and start correctly

Reply:
otherwise the checks would have failed

Reply:
Very good :+1: